docs/kyma-environment-broker/03-08-kyma-versions.md

```diff
@@ -45,7 +46,8 @@ You can also specify a Kyma version using the **kymaVersion** provisioning param
        },
        \"parameters\": {
            \"name\": \"$NAME\",
-           \"kymaVersion\": \"$VERSION"
+           \"kymaVersion\": \"$VERSION",
+           \"kymaPreviewVersion\": \"$PREVIEW_VERSION"
```

Comment:
the payload will be the same - do note change it here. We need to add a note that for preview plan we have separate default kyma version.